File: src/main/java/fr/d2factory/libraryapp/book/Book.java
package fr.d2factory.libraryapp.book;

/**
 * A simple representation of a book
 */
public class Book {
    String title;
    String author;
    ISBN isbn;

    public Book(String title, String author, ISBN isbn) {
        this.title = title;
        this.author = author;
        this.isbn = isbn;
    }
}

File: Csharp/LivreEmpruntés/Book/Book.cs
namespace fr.d2factory.libraryapp.book
{
	/// <summary>
	/// A simple representation of a book
	/// </summary>
	public class Book
	{
        public Book(string title, string author, Isbn isbn)
        {
            Title = title;
            Author = author;
            Isbn = isbn;
        }

        public string Title { get; protected set; } 
        public string Author { get; protected set; } 
        public Isbn Isbn { get; protected set; } 
         
	}

}
